src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -682,42 +689,45 @@ public boolean close(Channel channel)
 
         if (removed)
         {
-            if (channel == sctpConnection)
-            {
-                sctpConnection = null;
-            }
-
-            if (channel == channelForDtls)
+            synchronized (channelForDtlsLock)
```

Comment:
Again - the diff here looks bigger on GitHub's tool than it may appear - this is mainly whitespace due to being inside a synchronized block.